src/handler.ts

```diff
@@ -127,6 +140,7 @@ export default class DataHandler {
       } catch (err) {
         Logger.getInstance().error(`Operator ${name} failed for ${path} at step ${i}`,
           path);
+        // @eslint-disable-next-line
         console.error(err.message);
```

Comment:
Did the console log sneak in or do we need it?  If the latter,  should we `Logger.getInstance().error`?

Reply:
I'm not sure when that error arrived in the code but it doesn't seem necessary to me. I'll delete.